### ISSUE TITLE
Add BindingDB Data?

### ISSUE THREAD

**Issue by mkgilson** [deepchem/deepchem#2976]
How about creating a BindingDB dataset, or a BindingDB-minus-ChEMBL dataset?  

The BindingDB protein-ligand dataset includes the relevant data from ChEMBL, but also hundreds of thousands of measured protein-ligand binding data curated by BindingDB and not present in ChEMBL (unless they have recently stepped up their ingestion of our data). About 750K come from US patents. Note that SureChEMBL has compounds but not affinities.

If you'd like assistance from BindingDB staff identifying and ingesting our data, we'd be happy to help.

Mike

**Comment by rbharath** [comment]
Yes that would be great! If you would like to take a crack at adding a BindingDB dataset please go ahead.
